source/developer-guides/configuration.rst

```diff
@@ -24,53 +18,61 @@ Internal configuration data
-         :alt: Install Ubuntu 1 of 5
+.. image:: /images/ConfigFlow.png
+
+Configuration Editor Tool
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+|SPN| supports a Configuration Editor Tool (ConfigEditor.py) to configure firmware settings with graphics UI. This tool is included in |SPN| source package at /SblOpen/BootloaderCorePkg/Tools.
+
+.. image:: /images/CfgEditOpen.png
+
 
-_DSC File
+YAML Files
 ^^^^^^^^^^^^
-* DSC file is the heart of all platform settings including memory, silicon, GPIO, OS boot policy, security settings etc.
-* DSC file, in general is located in project specific board folder. 
-* For example, you can find DSC file for APL platform under SblOpen\Platform\ApollolakeBoardPkg\CfgData folder.
+* All platform configuration settings including memory, silicon, GPIO, OS boot policy, security settings etc are declared in a custom format and uses YAML Syntax.
+* YAML configuration files, in general are located in project specific board folder. 
+* For example, you can find the configuration files for Apollo Lake platform under SblOpen\Platform\ApollolakeBoardPkg\CfgData folder.
 
-Please note that you may find many DSC files. However, only CfgDataDef.dsc is the primary file used for the platform configuration, and other sub DSC files will be included by the primary DSC file to provide component specific configuration.
+Please note that you may find many YAML files. However, only CfgDataDef.ya is the primary file used for the platform configuration, and other sub YAML files will be 
+included by the primary YAML file to provide component specific configuration.
 
-The main platform configuration file is specified in CfgDataDef.dsc.
-The following screen shots will help explain. Once DSC file is loaded, all other grayed menu will be enabled.
+The main platform configuration file is specified in CfgDataDef.yaml.
+The following screen shots will help explain. Once an YAML file is loaded, all other grayed menu will be enabled.
 
-.. image:: /images/Config3.jpg
+.. image:: /images/CfgEditDefYaml.png
 
-_DLT File
+
+DLT Files
 ^^^^^^^^^^^^^
 
-* DLT (delta) file is used to provide overrides to DSC file to address board-level difference, including GPIO, boot policy, PCIE configuration, security settings etc.
+* DLT (delta) files are used to provide overrides to settings in YAML files to address board-level differences, including GPIO, boot policy, PCIE configuration, security settings etc.
 * DLT file contains unique Platform ID, and build tools will apply the settings to firmware images based on the platform ID.
+* DLT file that overrides configuration parameters for all boards (board id 0) is also supported. A typical use case is in case of Platform ID as explained below.
 
 DLT file can be generated in different ways:
 
 * Change any existing settings, and save it to DLT file with Configuration Editor Tool.
 * Load values from an existing binary file, and then save the changes as DLT file. 
 * Update existing DLT file with other text editor.

```

Comment:
Modifying through other editor is error-prone. In general, it is not recommended unless user is really family with it.  Maybe remove this ?